librz/core/cheap.c

```diff
@@ -0,0 +1,23 @@
+#include <rz_core.h>
+#include "core_private.h"
+
+/* API calls of windows heap for Cutter */
+#if __WINDOWS__
+RZ_API RzList *rz_heap_windows_blocks_list(RzCore *core) {
```

Comment:
Please add if the list owned or borrowed by the caller: https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#custom-pointer-modifiers